### PR TITLE
Thermal Daemon configuration for TGL Platform

### PR DESCRIPTION
**Pull request by padmashree9110**
Modified xml file to add configuration for TGL platform.

Tracked-On: OAM-99252
Signed-off-by: Balaji M m.balaji@intel.com
Signed-off-by: Pandey Prabhat Chand prabhat.chand.pandey@intel.com
Signed-off-by: Mandri Padmashree padmashree.mandri@intel.com

### CODE REVIEW

File: thermal/thermal-daemon/thermal-conf.xml [hunk 1]
```diff
@@ -1,6 +1,118 @@
 <?xml version="1.0"?>
 <!-- BEGIN -->
 <ThermalConfiguration>
+	<Platform>
+		<Name> TGL NUC i5  </Name>
+		<ProductName> NUC11PAHi5 </ProductName>
```
Comment (prabhatchand) [inline thermal/thermal-daemon/thermal-conf.xml:6]: there should not be any space before and after product name.
<ProductName>NUC11PAHi5</ProductName>

File: thermal/thermal-daemon/thermal-conf.xml [hunk 2]
```diff
@@ -1,6 +1,118 @@
...
+				<MinState>13000000</MinState>
+				<ReadBack>0</ReadBack>
+				<MaxState>6000000</MaxState>
+				<IncDecStep>-200000</IncDecStep>
+			</CoolingDevice>
+		</CoolingDevices>
+	</Platform>
+	<Platform>
+		<Name> TGL NUC i7  </Name>
+		<ProductName> NUC11PAHi7 </ProductName>
```
Comment (prabhatchand) [inline thermal/thermal-daemon/thermal-conf.xml:62]: there should not be any space before and after product name.
<ProductName>NUC11PAHi7</ProductName>

**Review by prabhatchand — APPROVED** [review]
Looks good to me.

**Review by prabhatchand — APPROVED** [review]
Looks good to me.

**Review by balajim001 — APPROVED** [review]
LGTM

**Review by balajim001 — APPROVED** [review]
LGTM

**Pull request merged**